config.js

```diff
@@ -62,6 +62,11 @@ const ConfigOptions = () => {
         },
 
         /**
+         * Copy takes an object of assets to copy from src to dist folders
+         */
+        copy: {},
```

Comment:
Yeh – weirdly it did work fine.

Should I add in a line to that function as well?  Or alternatively might mean that the other lines aren’t needed potentially…